lib/web_ui/lib/src/engine/text_editing/text_editing.dart

```diff
@@ -120,9 +153,8 @@ class EngineAutofillForm {
             textCapitalization: TextCapitalizationConfig.fromInputConfiguration(
                 field['textCapitalization']));
 
-        // The focused text editing element will not be created here.
-        final AutofillInfo focusedElement =
-            AutofillInfo.fromFrameworkMessage(focusedElementAutofill);
+        ids.write(autofill.uniqueIdentifier);
```

Comment:
Should we put some kind of separator between the ids?

Reply:
done, added `*` as seperator.